HerPublicWebsite.UnitTests/BusinessLogic/Services/DateHelperTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using HerPublicWebsite.BusinessLogic.Services.RegularJobs;
+using NUnit.Framework;
+
+namespace Tests.BusinessLogic.Services;
+
+public class DateHelperTests
+{
+    [Test]
+    public void StartOfMonthService_WhenCalledWithoutParameter_ReturnsAPreviousDate()
```

Comment:
You haven't renamed this from `StartOfMonthService_` to `DateHelper_`, however the correct thing to do is to actually just remove that from the name of the test.

We are already in `DateHelperTests` class, so don't need to respecify that in each test name. For example, the tests in `GovUkApiTests` don't all start with `GovUkApi_`.